packages/samples/react/src/components/FormWrap.tsx

```diff
@@ -1,31 +1,27 @@
-import type { FC } from 'react';
 import React from 'react';
 
 import { KolButton, KolForm } from '@public-ui/react';
-import type { RefFormComponent } from './types';
 
-type FormWrapProps = {
-	RefComponent: RefFormComponent;
-} & Record<string, unknown>;
-
-export const FormWrap: FC<FormWrapProps> = (props) => (
-	<KolForm>
-		<div className="grid gap-4">
-			<props.RefComponent {...props} />
-			<div className="flex gap-4">
-				<KolButton _label="Submit" _icons="codicon codicon-arrow-right" _type="submit" _variant="primary" />
-				<KolButton
-					_label="Bunte Icons"
-					_icons={{
-						left: { icon: 'codicon codicon-heart-filled', style: { color: '#cc006e' } },
-						right: { icon: 'codicon codicon-squirrel', style: { color: '#b41b1b' } },
-					}}
-					_type="submit"
-					_variant="secondary"
-				/>
-				<KolButton _label="Reset" _type="reset" _variant="tertiary" />
-				<KolButton _label="Help" _type="button" _variant="ghost" />
+export const FormWrap = <P,>({ RefComponent: Component, ...props }: { RefComponent: any } & Partial<P>) => {
```

Comment:
Mit dem `RefComponent: any` bin ich nicht glücklich, habe aber (in einer sinnvollen Zeit) keine bessere Lösung gefunden. Gerne Input geben :)

Reply:
Da die Typen der Komponenten in der `_types` Property Konflikte haben ist es wohl nicht besser lösbar. Für mich in Ordnung.